Dockerfile-auth

```diff
@@ -67,16 +67,11 @@ COPY dockerdata/startup.py /usr/local/sbin/pdns_server-startup
 # Config file(s) from builder
 # Should not grab this from builder - since it isn't being built
 COPY --from=builder /source/dockerdata/pdns.conf /etc/powerdns/
-RUN mkdir -p /etc/powerdns/pdns.d /var/run/pdns
-RUN touch /etc/powerdns-api.conf && chown 953 /etc/powerdns-api.conf
-RUN ln -s /etc/powerdns-api.conf /etc/powerdns/pdns.d/api.conf
```

Comment:
why remove this?

Reply:
startup.py now takes care of writing the API config if needed (I'll check)

Reply:
I believe that, but this construct was designed so that startup.py would not have to write into /etc/powerdns just to set an API key. I understand that for startup.py's templating, this is expected, but just for the API key, somebody mentioned that I should not be writing into somebody's bindmounted /etc/powerdns.

Reply:
The use should not set that env var then :). We won't write into the config dir then

Reply:
But then they cannot set the API key..

Reply:
if their bind-mounting.... they can

Reply:
I think we're fine actually!

If a user bind-mounts over `/etc/powerdns` or `/etc/powerdns/conf.d` (or the equivs for the rec and dnsdist) and does not set the TEMPLATE_FILES or `*_API_KEY` variables, the startup will never write to the config dirs